locales/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-gettext_templates(${CMAKE_CURRENT_SOURCE_DIR} ${ALL_SOURCES})
-gettext_compile(${CMAKE_CURRENT_SOURCE_DIR} share/locale)
-SET_DIRECTORY_PROPERTIES(PROPERTIES CLEAN_NO_CUSTOM TRUE)
-export_var(GETTEXT_ENABLED)
+if (NOT WIN32)
+    gettext_templates(${CMAKE_CURRENT_SOURCE_DIR} ${ALL_SOURCES})
```

Comment:
Yes, I just filed FACT-1500 for this. To me, it made sense to file this under Facter, even though much
of the fix will happen in Leatherman. The notes should make this clear.